### PR TITLE
Change templates importing to use Assets collection

### PR DESCRIPTION
**Pull request by kieckhafer**
Fixes the issue where Templates were not loaded on initial start up.

Will close #1563 as this is a replacement for what was happening in there.

### CODE REVIEW

**Comment by aaronjudd** [comment]
Getting an error.. rrr
```
=> Errors prevented startup:

   While processing files with ecmascript (for target os.osx.x86_64):
   server/api/core/templates.js:189:16: /server/api/core/templates.js:
   Duplicate declaration "Templates"

=> Your application has errors. Waiting for file change.
~```

File: server/api/core/templates.js [hunk 1]
```diff
@@ -3,7 +3,8 @@ import ReactDOMServer from "react-dom/server";
 import Handlebars from "handlebars";
 import Immutable from "immutable";
 import { Import } from "./import";
-import { Templates } from "/lib/collections";
+import { Assets, Shops, Templates } from "/lib/collections";
```
Comment (aaronjudd) [inline server/api/core/templates.js]: I'm seeing linting errors, `Import`, `Shops` not used.

`export function Templates() {` already defined.

**Comment by kieckhafer** [comment]
Renamed `Templates` function so it's not a duplicate.

**Comment by aaronjudd** [comment]
Testing latest linted update, `reaction reset + modules reinstall`. Just getting loader spinning when viewing PDP example.

**Comment by aaronjudd** [comment]
👍  Looks good to me.

**Comment by aaronjudd** [comment]
Possibly related, not seeing templates in the UI table. (but the collection is populated)

**Comment by aaronjudd** [comment]
ok, really mean it this time. looks good, merge.

**Comment by zenweasel** [comment]
Yeah, second what @hrath2015 said but otherwise a go

**Pull request merged**